packages/atomic/src/components/facets/atomic-segmented-facet-scrollable/atomic-segmented-facet-scrollable.tsx

```diff
@@ -18,16 +43,21 @@ export class AtomicSegmentedFacetScrollable {
           style="square-neutral"
           class="flex shrink-0 basis-8 justify-center items-center rounded"
           ariaHidden="true"
+          onClick={() => this.scrollLeft.emit()}
         >
           <atomic-icon class="w-3.5" icon={ArrowLeftIcon}></atomic-icon>
         </Button>
-        <div class="wrapper-segmented flex flex-row overflow-x-scroll">
+        <div
+          id="horizontalScroll"
```

Comment:
use a ref here https://stenciljs.com/docs/templating-jsx#getting-a-reference-to-a-dom-element
We do it in some other components too if you need some examples